[2022-03-28 10:13:32,532] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2022-03-28T16:29:10.852578+00:00 [queued]>
[2022-03-28 10:13:32,538] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2022-03-28T16:29:10.852578+00:00 [queued]>
[2022-03-28 10:13:32,540] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-28 10:13:32,540] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-28 10:13:32,540] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-28 10:13:32,551] {taskinstance.py:1264} INFO - Executing <Task(BashOperator): run_after_loop> on 2022-03-28 16:29:10.852578+00:00
[2022-03-28 10:13:32,553] {standard_task_runner.py:52} INFO - Started process 52719 to run task
[2022-03-28 10:13:32,564] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', 'manual__2022-03-28T16:29:10.852578+00:00', '--job-id', '13', '--raw', '--subdir', '/home/vengat/big_data/python/venv/aflow_env/lib/python3.9/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmplorm5861', '--error-file', '/tmp/tmp4dmqw1pn']
[2022-03-28 10:13:32,565] {standard_task_runner.py:77} INFO - Job 13: Subtask run_after_loop
[2022-03-28 10:13:32,618] {logging_mixin.py:109} INFO - Running <TaskInstance: example_bash_operator.run_after_loop manual__2022-03-28T16:29:10.852578+00:00 [running]> on host ubuntu
[2022-03-28 10:13:32,660] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2022-03-28T16:29:10.852578+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-03-28T16:29:10.852578+00:00
[2022-03-28 10:13:32,660] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-28 10:13:32,661] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 1']
[2022-03-28 10:13:32,668] {subprocess.py:85} INFO - Output:
[2022-03-28 10:13:32,669] {subprocess.py:89} INFO - 1
[2022-03-28 10:13:32,669] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-28 10:13:32,688] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20220328T162910, start_date=20220328T171332, end_date=20220328T171332
[2022-03-28 10:13:32,740] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-28 10:13:32,781] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
